plugins/org.jkiss.dbeaver.ext.postgresql.ui/src/org/jkiss/dbeaver/ext/postgresql/tools/PostgreBackupWizardPageObjects.java

```diff
@@ -333,9 +344,28 @@ public void saveState() {
         }
     }
 
+    private boolean isAllSchemaSelected() {
+        boolean allSchemasSelected = true;
+        for (TableItem item : schemasTable.getItems()) {
+            if (!item.getChecked()) {
+                allSchemasSelected = false;
+                break;
+            }
+        }
+        return allSchemasSelected;
+    }
+    private void updateCompleteBackupState() {
+    	boolean allSchemasSelected =isAllSchemaSelected();
+        completeBackupCheck.setEnabled(allSchemasSelected);
+        if (!allSchemasSelected) {
+            completeBackupCheck.setSelection(false);
+            wizard.getSettings().setCompleteBackup(false);
+        }
```

Comment:
Does it update correctly when nodes get checked and unchecked? To me, it would make more sense to use `allSchemasSelected` instead of `false` and remove that `if` completely.